tests/unit/_internal/utils/test_pickle.py

```diff
@@ -23,15 +23,15 @@ def test_pep574_restore() -> None:
     indices: list[int]
     bs, concat_buffer_bs, indices = pep574_dumps(lst)
     restored = t.cast(
-        list["ext.NpNDArray"], pep574_loads(bs, concat_buffer_bs, indices)
+        t.List["ext.NpNDArray"], pep574_loads(bs, concat_buffer_bs, indices)
```

Comment:
what is the change here, you can just quote the type `"list['ext.NpNDArray']"` for `t.cast` right?